docs/core/stories.rst

```diff
@@ -49,13 +52,73 @@ This is what we call a **story**.
   in the format ``intent{"entity1": "value", "entity2": "value"}``.
```

Comment:
Remove line 43 as we've already introduced the name `story` -- replace it with an intro to the bullet points, e.g. "A few things to note about stories:" or something like that